reference/speeduino.ini

```diff
@@ -286,7 +286,7 @@ page = 1
 
       ; Config1
       mapSample     = bits,   U08,      36, [0:1], "Instantaneous", "Cycle Average", "Cycle Minimum", "Event Average"
-      twoStroke     = bits,   U08,      36, [2:2], "Four-stroke", "Two-stroke"
+      strokes     = bits,   U08,      36, [2:2], "Four-stroke", "Two-stroke"
```

Comment:
Same as with `alternate` above. This is a reserved variable name and must be kept (unfortunately)

Reply:
Fixed